### PR TITLE
Add step to clean urls, not sure why one UT doesn't work

### PR DESCRIPTION
**Pull request by huyqd**

### CODE REVIEW

File: tests/test_DAG_url_scraper.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        mongo_database = MongoDb()
+        mongo_database._database = mongomock.MongoClient()["newspaper"]
+
+        # when
+        mongo_database.insert_tasks(tasks, "LANGUAGE")
+
+        # then
+        article_count = mongo_database._database["newspaper"]["LANGUAGE"].count_documents({})
+        assert article_count == 2
```
Comment (huyqd) [inline tests/test_DAG_url_scraper.py:43]: @nikolasrieble I have no idea why this test doesn't work, could you have a look?

Comment (nikolasrieble) [reply tests/test_DAG_url_scraper.py:43]: I would prefer to have two separate tests, each testing a variant. One time for stripping https, one time for stripping #

Comment (nikolasrieble) [reply tests/test_DAG_url_scraper.py:43]: But sure, will have a look at it.

File: tests/test_DAG_url_scraper.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+from unittest import TestCase
+from dags.DAG_url_scraper import get_clean_urls
+
+
+class Test(TestCase):
+
+    def test_get_clean_tasks(self):
```
Comment (nikolasrieble) [inline tests/test_DAG_url_scraper.py:7]: imho a test should only test a specific behaviour / part of the logic. In that way you directly know where you have a problem - if the test fails. Better split this general test into multiple separate ones.

**Comment by huyqd** [comment]
@nikolasrieble can we merge this PR?

**Review by nikolasrieble — APPROVED** [review]
Sure, looks good.

**Pull request merged**